x/operator/client/cli/query.go

```diff
@@ -225,3 +228,93 @@ func GetAllOperatorConsAddrs() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// QueryOperatorUSDValue queries the opted-in USD value for the operator
+func QueryOperatorUSDValue() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryOperatorUSDValue operatorAddr avsAddr",
+		Short: "Get the opted-in USD value for the operator",
+		Long:  "Get the opted-in USD value for the operator",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryOperatorUSDValueRequest{
+				Details: &operatortypes.OperatorAVSAddressDetails{
+					OperatorAddr: args[0],
+					AVSAddress:   args[1],
+				},
+			}
+			res, err := queryClient.QueryOperatorUSDValue(context.Background(), req)
+			if err != nil {
+				return err
+			}
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+	return cmd
+}
+
+// QueryAVSUSDValue queries the USD value for the avs
+func QueryAVSUSDValue() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryAVSUSDValue avsAddr",
+		Short: "Get the USD value for the avs",
+		Long:  "Get the USD value for the avs",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryAVSUSDValueRequest{
+				AVSAddress: args[0],
+			}
+			res, err := queryClient.QueryAVSUSDValue(context.Background(), req)
+			if err != nil {
+				return err
+			}
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+	return cmd
+}
+
+// QueryOperatorSlashInfo queries the slash information for the specified operator and AVS
+func QueryOperatorSlashInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryOperatorUSDValue operatorAddr avsAddr",
+		Short: "Get the the slash information for the operator",
+		Long:  "Get the the slash information for the operator",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryOperatorSlashInfoRequest{
+				Details: &operatortypes.OperatorAVSAddressDetails{
+					OperatorAddr: args[0],
+					AVSAddress:   args[1],
```

Comment:
Please validate these inputs

Reply:
same as above comment

Reply:
This result should be paginated. I believe this one will silently fail if the inputs are malformed.

Reply:
Yeah, thanks